src/launch/launch.scss

```diff
@@ -1,3 +1,110 @@
 @import "styles/colors.scss";
+@import "styles/styles.scss";
 .page.launch {
+  display: grid;
+  justify-content: center;
+  grid-gap: 30px;
+  .banner {
+    .banner1 {
+      // Make like a heading
+      @extend .heading;
+      font-size: 4vw;
+      display: grid;
+      justify-content: center;
+    }
+    // TODO: Customize for mobile
+    .banner2 {
+      @extend .subtext;
+      // TODO: Get font from figma
+      font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
```

Comment:
limit yourself to the two fonts we have been using: Aeonik and Cera Pro

Reply:
Aha, I wanted to change this anyways